python/generate/templates/rest.mustache

```diff
@@ -75,52 +74,26 @@ class RESTClientObject(object):
                 maxsize = 4
 
         # tcp_keep_alive cannot be put in additional_pool_args without errors in the base pool manager
-        if configuration.proxy:
-            if configuration.tcp_keep_alive:
-                self.pool_manager = TCPKeepAliveProxyManager(
-                    num_pools=pools_size,
-                    maxsize=maxsize,
-                    cert_reqs=cert_reqs,
-                    ca_certs=ca_certs,
-                    cert_file=configuration.cert_file,
-                    key_file=configuration.key_file,
-                    proxy_url=configuration.proxy,
-                    proxy_headers=configuration.proxy_headers,
-                    **addition_pool_args
-                )
-            else:
-                self.pool_manager = urllib3.ProxyManager(
```

Comment:
As you have done for the PoolManager below, we still need proxy support when the `configuration.pool_manager_fn` does not exist i.e. when the user is not using the ApiClientBuilder.

Reply:
Makes sense, have updated